docs/getting-started/creating-a-new-application-with-the-us-forms-system-starter-app.md

```diff
@@ -0,0 +1,7 @@
+# Creating a new application with the US Forms System Starter App
+
+The US Forms System Starter App creates the initial files, configuration, build process, and web server you need to start building your form immediately.
+
+For more information, including app installation instructions, see the [US Forms System Starter App README](https://github.com/usds/us-forms-system-starter-app#us-forms-system-starter-app).
```

Comment:
I was talking to @fatimanoorva this afternoon about where to put the instructions for setting up the starter app. I'm leaning toward wanting to keep the instructions in this repo alongside the other docs, rather than keeping two sets of docs in separate repos. Thoughts/feelings?

There's also an issue open about needing to enhance those instructions (#79). I'm working on getting the starter app running on my machine, too, and taking notes.